resources/instructions/createSObjectLwcQuickActions.html

```diff
@@ -35,76 +35,76 @@
 <body>
     <h1>Create sObject LWC Quick Actions</h1>
 
-    <p>
-        The following sObjects are present in your configured landing page:
-    </p>
-
-    <table id="quickActionStatusTable">
-        <thead>
-            <tr>
-                <th rowspan="2">sObject</th>
-                <th colspan="3">LWC Quick Actions</th>
-            </tr>
-            <tr>
-                <th>view</th>
-                <th>edit</th>
-                <th>create</th>
-            </tr>
-        </thead>
-        <tbody>
-            <!-- will be filled in programatically below -->
-        </tbody>
-    </table>
-
-    <label style="display: block; margin-top: 10px; margin-bottom: 8px" for="generateLwcQuickActionsButton">
-        Click 'Generate LWC Quick Actions' to create the missing LWC Quick
-        Actions:
-    </label>
-    <button id="generateLwcQuickActionsButton">
-        Generate LWC Quick Actions
-    </button>
-    <script>
-        const generateQuickActionsButtonElement = document.getElementById(
-            'generateLwcQuickActionsButton'
-        );
-        generateQuickActionsButtonElement.addEventListener('click', () => {
-            webviewMessaging.sendMessageRequest(
+        <p id="sobjectDiv">
```

Comment:
Looking through this, I'm realizing that all of the HTML files under `resources/instructions` have nominally been put through `prettier` formatting already. Let's codify that as part of this PR:

- Run `prettier` on this file, to fix up spacing, strings, etc.
- Amend `prettier:verify` in `package.json` to add the `"resources/instructions/**/*.html"` path.
- Amend `prettier:write` to do the same.

We may as well try to keep these formatted too, since they mostly already are.

Reply:
I think this is a great idea. HOWEVER, can we please not do it in this PR because I have stacked changes on this PR and will have to go through even more merge conflicts if we do. I can add this in my next PR?

Reply:
Oh yeah sure. No need to blow up the train as part of the process!

Reply:
Noted. I will make another PR for `prettier` after this gets merged.